src/lightning_app/utilities/cli_helpers.py

```diff
@@ -119,3 +121,15 @@ def _retrieve_application_url_and_available_commands(app_id_or_name_or_url: Opti
                     raise Exception(f"The server didn't process the request properly. Found {resp.json()}")
                 return lightningapp.status.url, _extract_command_from_openapi(resp.json())
     return None, None
+
+
+def _arrow_time_callback(_ctx, _param, value, arw_now=arrow.utcnow()) -> arrow.Arrow:
```

Comment:
Just asking: do we have UTC on all timestamps in all system? Or this translation might cause confusion in some cases?

Reply:
Yes, we use UTC, and this is a good point :) But the confusion is a well-known for many cases.